var/spack/repos/builtin/packages/protobuf/package.py

```diff
@@ -68,10 +75,36 @@ def fetch_remote_versions(self):
 
     def cmake_args(self):
         args = [
+            '-DCMAKE_INSTALL_PREFIX=%s' % self.prefix,
```

Comment:
You'll want to add `std_cmake_args` in your `install` method, then you can undo this single line change